### DISCUSSION TITLE
Tracing for reactive drivers?

### DISCUSSION THREAD

**Discussion by edeandrea** [quarkusio/quarkus#22651] [Q&A] [answered]
The [Quarkus OpenTracing Guide](https://quarkus.io/guides/opentracing#jdbc) shows support for tracing JDBC queries. Is there anything similar available for the reactive drivers?

[a comment by an automated account is not shown]

**Comment by radcortez** [comment, marked as answer]
I'm not aware, but any related/required work will happen in OpenTelemetry.

**Comment by kavishkamk** [comment]
Hey. Can I know about the OpenTelemetry use with reactive-driver. I am try to integrate quarkus application with OpenTelemetry and it use reactive-driver with liquiebase .

**Reply by edeandrea** [reply]
It just works. The reactive drivers will create spans if they detect OpenTelemetry on the classpath.

The [Hero service in the Quarkus Superheroes](https://github.com/quarkusio/quarkus-super-heroes/tree/main/rest-heroes) uses the reactive postgres driver with OpenTelemetry.
